lib/meta-boxes.php

```diff
@@ -221,4 +221,217 @@ function igv_cmb_metaboxes_locate_page() {
   //
 
 }
+
+/**
+ * Add Vodka page metaboxes
+ */
+add_action( 'cmb2_init', 'igv_cmb_metaboxes_vodka_page' );
+function igv_cmb_metaboxes_vodka_page() {
+
+  $prefix = '_igv_';
+
+  // Get Locate page
+  $vodka_page = get_page_by_path('/vodka');
+
+  // Check if vodka page exists
+  if(!empty($vodka_page)) {
+
+    // Vodka Init metabox
+    $vodka_meta = new_cmb2_box( array(
+      'id'            => $prefix . 'vodka_options',
+      'title'         => __( 'Vodka Options', 'cmb2' ),
+      'object_types'  => array( 'page', ), // Post type
+      'show_on'       => array(
+        'key' => 'id',
+        'value' => array($vodka_page->ID)
+      ),
+      'context'       => 'normal',
+      'priority'      => 'high',
+      'show_names'    => true, // Show field names on the left
+    ) );
+
+    //Vodka Video group
+    $vodka_video_id = $vodka_meta->add_field( array(
+      'id'      => $prefix . 'vodka_video_group',
+      'name'    => 'Video',
+      'type'        => 'group',
+      'repeatable'  => false, // use false if you want non-repeatable group
+    ) );
+
+    //Vodka Video mp4
+    $vodka_meta->add_group_field( $vodka_video_id, array(
+      'name'    => 'Video (mp4)',
+      'id'      => $prefix . 'vodka_video_mp4',
```

Comment:
Fields added to groups don't need to use the prefix. This data get saved something like 
```
array(
  'prefixed_group_id' => array(
    'field_1' => 'somthing',
    'field_2' => 'somthing else'
  )
)
```

Since the group `id` is already prefixed there won't be any issues with data clashing. Ex. two fields with the same id but in different groups.

Reply:
`mp4`, `webm`, `poster`